src/apps/expenses/components/Expense.js

```diff
@@ -268,17 +268,21 @@ class Expense extends React.Component {
         </style>
 
         <div className="fromCollective">
-          <a
-            href={`/${expense.fromCollective.slug}`}
+          <Link
+            route="collective"
+            params={{ slug: expense.fromCollective.slug }}
             title={expense.fromCollective.name}
+            passHref
```

Comment:
The `passHref` appears to be necessary given we use the wrapper Link component provided by `next-routes`. Otherwise we don't have working links that can open in new tabs or be copied.

Reply:
Under the hood [`next-routes` link is just a regular NextJS link](https://github.com/fridays/next-routes/blob/master/src/index.js) with the proper URL setup. It doesn't add extra markup. I understand the need of `passHref` when children is a `<MenuLink/>` of a `<StyledLink/>` for example, but not here.